Base/QTCore/qSlicerCoreApplication.cxx

```diff
@@ -1139,7 +1139,8 @@ void qSlicerCoreApplication::handleCommandLineArguments()
     scriptArgs.removeAll("--attach-process");
     if(!extraPythonScript.isEmpty())
       {
-      scriptArgs.removeFirst();
+      // Remove extra Python script processed arguments  (e.g -I /path/to/script.py)
+      scriptArgs.remove(0, options->extraPythonScriptProcessedArgumentsCount());
```

Comment:
My initial suggestion was an oversight and this is not working.

--- 

(I will suggest another approach after I get a power outlet for my laptop :battery: )

Reply:
```suggestion
      scriptArgs.erase(scriptArgs.begin(), scriptArgs.begin() + options->extraPythonScriptProcessedArgumentsCount());
```

Reply:
This should address the following:

```
 /usr/src/Slicer/Base/QTCore/qSlicerCoreApplication.cxx: In member function ‘virtual void qSlicerCoreApplication::handleCommandLineArguments()’:
/usr/src/Slicer/Base/QTCore/qSlicerCoreApplication.cxx:1143:18: error: ‘class QStringList’ has no member named ‘remove’; did you mean ‘removeAt’?
       scriptArgs.remove(0, options->extraPythonScriptProcessedArgumentsCount());
                  ^~~~~~
                  removeAt
```